test_scripts/iAP2TransportSwitch/common.lua

```diff
@@ -27,29 +27,28 @@ m.device = {
     id = "127.0.0.1",
     port = 23456,
     out = "iap2bt.out",
-    type = "BLUETOOTH"
+    type = "BLUETOOTH",
```

Comment:
@dev-gh Please remove unused module 'hmi_values' in line 17

Reply:
@dboltovskyi 
Done with 24aad0b